### PR TITLE
Inform WistiaPlayerViewControllerDelegate when end of video has been …

### PR DESCRIPTION
**Pull request by moger777**
…reached.

Fixes https://github.com/wistia/WistiaKit/issues/74

### CODE REVIEW

File: Pod/Classes/Playback/public/WistiaPlayerViewController.swift [hunk 1]
```diff
@@ -91,6 +91,13 @@ public protocol WistiaPlayerViewControllerDelegate : class {
      - Parameter vc: The `WistiaPlayerViewController` whose hosting app is entering the background.
     */
     func shouldContinuePlaybackWhenEnteringBackground(_ vc: WistiaPlayerViewController) -> Bool
+
+    /**
+     Informs delegate that end of video has been reached.
+
+     - Parameter vc: The `WistiaPlayerViewController` that finished playing video
+    */
+    func didPlayToEndTime(_ vc: WistiaPlayerViewController)
```
Comment (spinosa) [inline Pod/Classes/Playback/public/WistiaPlayerViewController.swift]: Let's make implementation of this optional by providing a default, do-nothing implementation in the protocol extension, below.

**Comment by moger777** [comment]
@spinosa just pushed an empty implementation to method for extension.

**Comment by moger777** [comment]
@spinosa thanks for merging that. Would it be possible to get these changes merged in the swift 4 branch also?

**Comment by spinosa** [comment]
done and done =)

https://github.com/wistia/WistiaKit/commit/c60675be6ba4e5febb88e2c9abcfd0104599a337#diff-365140f93973bebdc928e7f23e85febcR100

**Pull request merged**